src/crate/qa/tests.py

```diff
@@ -228,16 +237,22 @@ def setUp(self):
         self._log_consumers = []
 
         def new_node(version, settings=None, env=None):
-            self._path_data = self.mkdtemp()
```

Comment:
Why don't we just turn `self._path_data` in to a list and store each nodes generated temp path inside? this way, `tearDown` functionality would still work

Reply:
afaik, tearDown [removes](https://github.com/crate/crate-qa/blob/master/src/crate/qa/tests.py#L271) tmp dir's  created by `self.mkdtemp()`

Reply:
ah right, even better